docs/products/postgresql/howto/use-pg_qualstats.rst

```diff
@@ -0,0 +1,331 @@
+
+How it works
+------------
+
+The primary task of the pg_qualstats extension is to collect statistics on predicates in active PostgreSQL queries. pg_qualstats uses a distinctive method of gathering statistics on predicates in WHERE statements and JOIN clauses, as opposed to the traditional approach of statistically analyzing full queries based on the total execution time. The novel approach that pg_qualstats uses allows the identification of frequently executed predicates (``quals``), which constitute an input for identifying missing indexes and producing index creation recommendations.
+
+.. topic:: Flow of actions
+
+    1. You enable pg_qualstats on your service, database, or user (role).
+    2. You configure pg_qualstats parameters if the default values don't match your use case.
+    3. pg_qualstats starts analyzing active Aiven for PostgreSQL queries on a service, within a database, or for a user.
+    4. Based on the analysis, pg_qualstats produces and stores results by populating statistics and making them accessible for your queries.
+    5. You execute read-only functions and select from predefined views to retrieve insights on missing indexes or index creation suggestions.
+
+.. seealso::
+
+   For more information on how pg_qualstats works, check `powa-team's pg_qualstats <https://github.com/powa-team/pg_qualstats>`_.
+
+Prerequisites
+-------------
+
+* Aiven organization, project, and service (Aiven for PostgreSQL)
+* Depending on what interface you'd like to use for interacting with pg_qualstats, you might need the following:
+    * Access to `Aiven Console <https://console.aiven.io/>`_
+    * SQL
+    * `Aiven API <https://api.aiven.io/doc/>`_
+    * :doc:`Aiven CLI client </docs/tools/cli>`
+
+Enable pg_qualstats
+-------------------
+
+You can enable the extension in `Aiven Console <https://console.aiven.io/>`_, using SQL, Aiven API, or Aiven CLI.
+
+Enable in Aiven Console
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. important::
+
+   In `Aiven Console <https://console.aiven.io/>`_, you can enable pg_qualstats at the service level only. To enable the extension on a database or for a user, you need to use SQL.
+
+1. Log in to `Aiven Console <https://console.aiven.io/>`_ and navigate to a desired organization, project, and service.
+2. On the **Overview** page of your service, navigate to **Advanced configuration** > **Configure** > **Add configuration options**.
+3. Find the **pg_qualstats.enabled** parameter, set it to **true**, and select **Save configuration**.
+
+Enable with SQL
+~~~~~~~~~~~~~~~
+
+.. note::
+
+   SQL allows for fine-grained enablement of pg_qualstats: on a database, for a user (role), or for a database-role combination.
+
+Enable on a database
+''''''''''''''''''''
+
+1. :doc:`Connect to your Aiven for PostgreSQL service </docs/products/postgresql/howto/list-code-samples>`.
+
+2. Run the following query:
+
+   .. code-block:: bash
+
+      ALTER DATABASE DATABASE_NAME SET pg_qualstats.enabled = 'yes'
```

Comment:
Even though different values are valid for a boolean like this, the one used consistently in PostgreSQL documentation is on / off :

```
ALTER DATABASE DATABASE_NAME set pg_qualstats.enabled = 'on'
```